apps/api/src/form/id-generator.ts

```diff
@@ -0,0 +1,6 @@
+import { customRandom } from 'nanoid';
+
+export const generateConfirmationId = () =>
+  customRandom('0123456789ABCDEFGHIJKLMNPQRSTUVWXYZ', 13, size => {
```

Comment:
Possible remove `I` and/or `L` from this list? I ~ 1 ~ L

Reply:
These are all caps. L shouldn't be an issue. One risk though is reducing the options will increase the chance of collision.